build.gradle

```diff
@@ -155,4 +157,17 @@ configure(coreProjects) {
         }
     }
     check.dependsOn jacocoTestCoverageVerification
+}
+
+licenseReport {
+    configurations = ALL
+    excludeOwnGroup = true
+    excludeBoms = false
+    renderers = new TextReportRenderer()
+}
+
+task copyThirdParyReport(type: Copy) {
```

Comment:
The plugin should be creating a large text file which has all the dependencies. That one file should be the one that should overwrite the `THIRD-PARTY` file. This is copying all the `.txt` files. It should probably copy one specific file to the `THIRD-PARTY` file.

Reply:
Yeah, I was not sure whether to replace it or create a new one.